roles/password_policy/vars/secrets.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Secret passwords file
+
+# new password for the users in the power HMC as per the password policy
+password_policy_user_password:
+    passwd: "Abcde@12345"
```

Comment:
I suggest don't hard code the password, you can let the user to define them just mention as <TODO_password> something like this